client/me/connected-application-item/index.jsx

```diff
@@ -120,116 +111,103 @@ module.exports = React.createClass( {
-				<p className="connected-application-item__content-description">
-					{ connection.description }
+			<div>
+				<h2>{ this.translate( 'Application Website' ) }</h2>
+				<p>
+					<a
+						href={ safeProtocolUrl( URL ) }
+						onClick={ this.recordClickEvent( 'Connected Application Website Link' ) }
+						target="_blank"
+					>
+						{ safeProtocolUrl( URL ) }
+					</a>
 				</p>
-				<ul className="connected-application-item__ul">
-					<li className="connected-application-item__connection-detail">
-						<strong className="connected-application-item__connection-detail-title">
-							{ this.translate( 'Application Website' ) }
-						</strong>
-
-						<span className="connected-application-item__connection-detail-description">
-							<a
-								href={ safeProtocolUrl( connection.URL ) }
-								onClick={ this.recordClickEvent( 'Connected Application Website Link' ) }
-								target="_blank"
-							>
-								{ safeProtocolUrl( connection.URL ) }
-							</a>
-						</span>
-					</li>
-
-					<li className="connected-application-item__connection-detail">
-						{
-							this.translate( '{{detailTitle}}Authorized On{{/detailTitle}}{{detailDescription}}%(date)s{{/detailDescription}}', {
-								components: {
-									detailTitle: <strong className="connected-application-item__connection-detail-title" />,
-									detailDescription: <span className="connected-application-item__connection-detail-description" />
-								},
-								args: {
-									date: this.moment( connection.authorized ).format( 'MMM D, YYYY @ h:mm a' )
-								}
-							} )
-						}
-					</li>
 
+				{ this.translate( '{{detailTitle}}Authorized On{{/detailTitle}}{{detailDescription}}%(date)s{{/detailDescription}}', {
+					components: {
+						detailTitle: <h2 />,
+						detailDescription: <p className="connected-application-item__connection-detail-description" />
+					},
+					args: {
+						date: this.moment( authorized ).format( 'MMM D, YYYY @ h:mm a' )
+					}
+				} ) }
+				<div>
 					{ this.renderScopeMessage() }
+				</div>
 
-					<li className="connected-application-item__connection-detail">
-						<strong className="connected-application-item__connection-detail-title">
-							{ this.translate( 'Access Permissions' ) }
-						</strong>
-
-						{ connection.permissions.map( function( permission ) {
-							return (
-								<span
-									className="connected-application-item__connection-detail-description"
-									key={ 'permission-' + permission.name } >
-									{ permission.description }
-								</span>
-							);
-						}, this ) }
-					</li>
+				<h2>
+					{ this.translate( 'Access Permissions' ) }
+				</h2>
+				<ul className="connected-application-item__connection-detail-descriptions">
+					{ permissions.map( function( permission ) {
+						return (
+							<li key={ 'permission-' + permission.name } >
+								{ permission.description }
+							</li>
+						);
+					}, this ) }
```

Comment:
It looks like we are sending in `this` into the `map`, but since `this` isn't used inside of the map, it's pretty much unnecessary. In fact, if we wanted to take one further step beyond simply removing `this`, we could take out some noise and replace this with a fat-arrow function:

```js
{ permissions.map( ( { name, description } ) => (
	<li key={ `permission-${ name }` }>
		{ description }
	</li>
) }
```